diff -uNr gl4es/src/glx/glx.c gl4es.mod/src/glx/glx.c
--- gl4es/src/glx/glx.c	2021-10-13 23:49:59.663443188 +0800
+++ gl4es.mod/src/glx/glx.c	2021-10-13 23:48:57.699443233 +0800
@@ -1391,8 +1391,11 @@
     EGLNativeWindowType native_window = (EGLNativeWindowType)drawable; // Casting GLXDrawable to EGLNativeWindowType
     eglSurf = context->eglSurface = egl_eglCreateWindowSurface(eglDisplay, context->eglConfigs[0], native_window, attrib_list);
 #else
     eglSurf = context->eglSurface = egl_eglCreateWindowSurface(eglDisplay, context->eglConfigs[0], drawable, attrib_list);
 #endif
